Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedBackingStoreProxy.cpp

```diff
@@ -51,12 +51,18 @@ void CoordinatedBackingStoreProxy::Update::appendUpdate(float scale, Vector<uint
 
     // Remove any creations or updates previously registered for tiles that are going to be removed now.
     if (!m_tilesToCreate.isEmpty() || !m_tilesToUpdate.isEmpty()) {
+        Vector<uint32_t, 8> removalsToRemove;
```

Comment:
A little bit complex name :sweat_smile: ? And they are in the end tiles ids? What about createdTilesRemoved?